Ginger/GingerCoreTest/AutoPilot/APIModelsCompareTest.cs

```diff
@@ -0,0 +1,117 @@
+﻿using amdocs.ginger.GingerCoreNET;
+using Amdocs.Ginger.Common;
+using Amdocs.Ginger.CoreNET.Repository;
+using Amdocs.Ginger.Repository;
+using Ginger.Run;
+using GingerCore.Drivers.WebServicesDriverLib;
+using GingerCore.Platforms;
+using GingerCoreNET.Application_Models;
+using GingerCoreNET.SolutionRepositoryLib.RepositoryObjectsLib.PlatformsLib;
+using GingerTestHelper;
+using GingerWPF.ApplicationModelsLib.APIModels;
+using GingerWPF.WorkSpaceLib;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace UnitTests.NonUITests.AutoPilot
+{
+    /// <summary>
+    /// API Models Comparison Utility Test
+    /// </summary>
+    /// 
+
+    [TestClass]
+    public class APIModelsCompareTest
+    {
+        static GingerRunner mGR = null;
+        static string wsdlURL = @"http://ws.cdyne.com/delayedstockquote/delayedstockquote.asmx?wsdl";
+
+        [ClassInitialize]
+        public static void ClassInitialize(TestContext TC)
+        {
+            Reporter.ToLog(eLogLevel.DEBUG, "Creating the GingerCoreNET WorkSpace");
+            WorkSpaceEventHandler WSEH = new WorkSpaceEventHandler();
+            WorkSpace.Init(WSEH);
+
+            string TempSolutionFolder = TestResources.GetTestTempFolder(@"Solutions" + Path.DirectorySeparatorChar + "APIModelsComparisonUtilityTest");
+            if (Directory.Exists(TempSolutionFolder))
+            {
```

Comment:
1- remove the dependency on external WSDL, instead save 5-6 APIModels on the Test Resources and load them into 1/2 lists in the Class Init to simulate ExsitingAPIModels LearnedAPIModels
2- instead 1 Unit test which test the all compare process, break it down into small unit tests which will test the building blucks functionalities like Match/Compare/etc. from each have several unit tests to test diffrent outcome